sources/core/objects/Grille.cpp

```diff
@@ -37,15 +37,36 @@ void Grille::AttackBy(IdPtr<Item> item)
             SetPassable(D_ALL, Passable::FULL);
             cutted_ = true;
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
+            return;
         }
         else
         {
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
             Delete();
+            return;
         }
     }
+    Breakable::AttackBy(item);
```

Comment:
Well it gives 0 so I cannot get the name of attacker, but if I pass the human as AttackBy(owner) then a lot of code will be buggy and it isn't a good way.